openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         )
         self.runtime_id: str | None = None
         self.runtime_url: str | None = None
+        self.runtime_init_timeout = self.config.sandbox.remote_runtime_init_timeout
```

Comment:
probably don't need to set this as a member var--can just use `self.config.sandbox.remote_runtime_init_timeout`